ansible/roles/distributed_press/templates/nginx-static.j2

```diff
@@ -12,26 +12,36 @@ server {
 
   error_page 404 /404.html;
 
+  add_header Cache-Control no-cache;
```

Comment:
Why no cache?

Reply:
> If you want caches to always check for content updates while reusing stored content, no-cache is the directive to use. It does this by requiring caches to revalidate each request with the origin server.
> 
> Note that no-cache does not mean "don't cache". no-cache allows caches to store a response but requires them to revalidate it before reuse. If the sense of "don't cache" that you want is actually "don't store", then no-store is the directive to use.
> 
> [source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#directives)

in practice since nginx is adding etag headers to every static file, it'll return 304 for cached urls so you save the body bytes, but still need an open connection to the page.  it's better to guess how long you need to cache and get asked why a very very very recent change doesn't show up yet.

Reply:
i made edits to the above comment